service/build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
     implementation "com.google.guava:guava:31.0.1-jre"
     implementation "io.prometheus:simpleclient_httpserver:0.12.0"
     implementation "io.opencensus:opencensus-exporter-stats-prometheus:0.28.3"
-    implementation "org.broadinstitute.cromwell:cromwell-client_2.13:0.1-7407120a2-SNAP"
+    implementation "org.broadinstitute.cromwell:cromwell-client_2.13:0.1-0854d6c79-SNAP"
```

Comment:
Oh, maybe not... it would be good to match a git hash from the cromwell `develop` branch's commit list, which presumably will only be there when the PR merges, just so we can track where on the timeline the particular hash we're using has come from

Reply:
Okay, I'll update once the Cromwell PR merges.